qiskit/qasm2/export.py

```diff
@@ -244,17 +244,18 @@ def _instruction_call_site(operation):
     else:
         qasm2_call = operation.name
     if operation.params:
-        qasm2_call = "{}({})".format(
-            qasm2_call,
-            ",".join([pi_check(i, output="qasm", eps=1e-12) for i in operation.params]),
-        )
+        qasm2_call = f"""{qasm2_call}({','.join(
+            [pi_check(i, output='qasm', eps=1e-12) for i in operation.params]
+        )})"""
```

Comment:
This might be cleaner with an additional variable, like you introduced something above